core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -1005,6 +1005,14 @@ package object config {
       .checkValue(v => v > 0, "The value should be a positive integer.")
       .createWithDefault(2000)
 
+  private[spark] val SHUFFLE_USE_OLD_FETCH_PROTOCOL =
+    ConfigBuilder("spark.shuffle.useOldFetchProtocol")
+      .doc("Whether to use the old protocol while doing the shuffle block fetching. " +
+        "It is only enabled while we need the compatibility in the scenario of new spark " +
+        "version job fetching blocks from old version external shuffle service.")
```

Comment:
`fetching blocks` -> `fetching shuffle blocks`

Reply:
Thanks, done in 4c4af33.